packages/generic-application/lib/server.js

```diff
@@ -0,0 +1,186 @@
+'use strict'
```

Comment:
no need for this, it's an esm file

Reply:
Leftover before I moved to ESM. Will remove this.